cves/kernel/CVE-2016-3137.yml

```diff
@@ -286,8 +285,8 @@ ipc:
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  answer:
-  note:
+  answer: false
+  note: 'This vulnerability does not include inter-process communication.'
```

Comment:
Isn't cypress_m8 considered IPC? I was curious how you came to this conclusion.

Reply:
You are right. Fixed